db/migrate/20201009142558_add_role_to_users.rb

```diff
@@ -0,0 +1,8 @@
+class AddRoleToUsers < ActiveRecord::Migration[6.0]
+  def change
+    add_column :users, :role, :string, null: false, default: ''
+    User.includes(:roles).all.each do |user|
```

Comment:
This may be a stupid question but how does this work if Rolify has been removed from the User models?

Reply:
Ignore - missed PR description.